### ISSUE TITLE
`prefer-text-field` should not flag `varchar` fields without a length specifier

### ISSUE THREAD

**Issue by reshleman** [sbdchd/squawk#137]
Hello!

The `prefer-text-field` rule flags `varchar` fields, with a note indicating that changing the length of a `varchar` field requires an `ACCESS EXCLUSIVE` lock.

However, this rule also flags `varchar` fields that do not have a length specifier, and therefore accept strings of any length:

```
$ squawk example.sql
example.sql:1:0: warning: prefer-text-field

   1 | CREATE TABLE IF NOT EXISTS sample(
   2 |   id serial PRIMARY KEY NOT NULL,
   3 |   something varchar
   4 | );

  note: Changing the size of a varchar field requires an ACCESS EXCLUSIVE lock.
  help: Use a text field with a check constraint.
```

_Adding_ a length limit to this field can be accomplished  without an `ACCESS EXCLUSIVE` lock using a `CHECK` constraint, just as with a `text` field, so I believe this rule should only be applied to `varchar` fields _with_ a length specifier (e.g., `varchar(2)`).

Thanks!

**Comment by sbdchd** [comment]
Thanks for the report, wasn't aware that a varchar without a length was allowed

**Comment by sbdchd** [comment]
Should be fixed in version 0.7.0 available on npm

```
❯ echo 'CREATE TABLE IF NOT EXISTS foo_tbl(bar_col varchar);' | squawk
Found 0 issues in 1 file 🎉
```

**Comment by reshleman** [comment]
🎉 Thanks for the speedy response + fix!
